changelog/0.39.md

```diff
@@ -81,3 +81,19 @@ Released 2024-07-15
 ### Other(s)
 
 - [#2187](https://github.com/elastisys/compliantkubernetes-apps/pull/2187) - bug: Fix bug preventing configuration of PSA level @simonklb
+
+## v0.39.2
```

Comment:
Should we mention what CVEs are remediated by the upgrades?

Reply:
I can add that but such things _should_ be added to the PRs to autogenerate such notices.

Reply:
Thanks, agree!